server/src/domain/auth/auth.service.ts

```diff
@@ -251,11 +251,26 @@ export class AuthService {
         storageLabel = null;
       }
 
+      let defaultQuota: number | null = config.oauth.defaultStorageQuota as keyof OAuthProfile as number;
+      if (typeof defaultQuota !== 'number') {
+        defaultQuota = null;
+      }
+
+      let storageQuota: number | null = profile[config.oauth.storageQuotaClaim as keyof OAuthProfile] as number;
+      if (typeof storageQuota !== 'number') {
```

Comment:
If the quota is a string it is ignored. I think you need to add some unit tests for this.

Reply:
A test that has a string in the claim and confirms it uses default quota?